validphys2/examples/pca_bias_variance_report.yaml

```diff
@@ -0,0 +1,50 @@
+meta:
+  title: PCA report for bias and variance of multiclosure fits.
+  author: Mark N. Costantini
+  keywords: [multiclosure, PCA, bias, variance]
+
+
+dataset_inputs:
+  - {dataset: ATLAS_SINGLETOP_7TEV_TCHANNEL-XSEC, variant: legacy}
```

Comment:
Why are you using the legacy versions? Is that necessary?

Reply:
The fits that I use in that runcard were done using th 200. I think this is the reason, why I added the legacy version